config/routes.rb

```diff
@@ -73,6 +74,7 @@
   get "/docs/tutorials/bitbucket",                to: redirect("/docs/integrations/bitbucket")
   get "/docs/tutorials/custom-saml",              to: redirect("/docs/integrations/sso/custom-saml")
   get "/docs/tutorials/sso-setup-with-graphql",   to: redirect("/docs/integrations/sso/sso-setup-with-graphql")
+  get "/docs/tutorials/2fa",                      to: redirect("/docs/team-management/2fa")
```

Comment:
In addition to adding the redirect I also suggest updating existing URLs to use the new path.

Reply:
As an aside (unrelated to this PR), I noted redirects are ignored on the [preview site](https://2483--bk-docs-preview.netlify.app/docs/tutorials/2fa), which makes sense.

Reply:
I reckon we could leave this 2FA tutorial out of the team-management, the benefit of leaving it under tutorials is that we can add the "enforce 2fa" documentation under team-management without it being confusing for readers having them side by side.

Reply:
> In addition to adding the redirect I also suggest updating existing URLs to use the new path.
Thanks @dannymidnight, I missed existing links, good catch and I'll push up a fix.